<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="markdown" source-language="en-US" target-language="nl-nl">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-192e1fd" tool-company="Microsoft" />
      <xliffext:olfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">4a88741020f6361d91ccb5eb9110d54a7c0d3afc</xliffext:olfilehash>
      <xliffext:olfilepath xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset\System.Net.NetworkInformation.PingException.yml</xliffext:olfilepath>
      <xliffext:oltranslationpriority xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset</xliffext:oltranslationpriority>
      <xliffext:oltranslationtype xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">Human Translation</xliffext:oltranslationtype>
      <xliffext:olskeletonhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">ce589c1a655feb469617c5b0ecb0179c27f9a122</xliffext:olskeletonhash>
      <xliffext:olxliffhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">838890aab49e881865c3043922fdc1c7cb36d97a</xliffext:olxliffhash>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve">
          <source>The exception that is thrown when a &lt;xref:System.Net.NetworkInformation.Ping.Send<bpt id="p1">*</bpt>&gt; or &lt;xref:System.Net.NetworkInformation.Ping.SendAsync<ept id="p1">*</ept>&gt; method calls a method that throws an exception.</source>
          <target state="translated">De uitzondering die wordt gegenereerd wanneer een &lt;&gt; &lt;/&gt; <bpt id="p1">*</bpt>&gt; of &lt;&gt; &lt;/&gt; <ept id="p1">*</ept>&gt; methode roept een methode die een uitzondering veroorzaakt.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT">
          <source>The &lt;xref:System.Net.NetworkInformation.Ping&gt; class throws this exception to indicate that while sending an Internet Control Message Protocol (ICMP) Echo request, a method called by the &lt;xref:System.Net.NetworkInformation.Ping&gt; class threw an unhandled exception.</source>
          <target state="translated">De &lt;xref:System.Net.NetworkInformation.Ping&gt;klasse genereert deze uitzondering om aan te geven dat tijdens het verzenden van een Internet Control Message Protocol (ICMP) echoaanvraag een methode aangeroepen door de &lt;xref:System.Net.NetworkInformation.Ping&gt;klasse heeft een niet-verwerkte uitzondering veroorzaakt.&lt;/xref:System.Net.NetworkInformation.Ping&gt; &lt;/xref:System.Net.NetworkInformation.Ping&gt;</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT">
          <source>Applications should check the inner exception of a PingException object to identify the problem.</source>
          <target state="translated">Toepassingen Controleer de binnenste uitzondering van een object PingException om het probleem te identificeren.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT">
          <source>The &lt;xref:System.Net.NetworkInformation.Ping&gt; class does not throw this exception if the ICMP Echo request fails because of network, ICMP, or destination errors.</source>
          <target state="translated">De &lt;xref:System.Net.NetworkInformation.Ping&gt;klasse genereert deze uitzondering niet als de ICMP-echoaanvraag vanwege fouten in het netwerk, ICMP- of doelserver mislukt.&lt;/xref:System.Net.NetworkInformation.Ping&gt;</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT">
          <source>For such errors, the &lt;xref:System.Net.NetworkInformation.Ping&gt; class returns a &lt;xref:System.Net.NetworkInformation.PingReply&gt; object with the relevant &lt;xref:System.Net.NetworkInformation.IPStatus&gt; value set in the &lt;xref:System.Net.NetworkInformation.PingReply.Status%2A&gt; property.</source>
          <target state="translated">Voor dergelijke fouten het &lt;xref:System.Net.NetworkInformation.Ping&gt;klasse retourneert een &lt;xref:System.Net.NetworkInformation.PingReply&gt;object met de relevante &lt;xref:System.Net.NetworkInformation.IPStatus&gt;waarde ingesteld in de &lt;xref:System.Net.NetworkInformation.PingReply.Status%2A&gt;eigenschap.&lt;/xref:System.Net.NetworkInformation.PingReply.Status%2A&gt; &lt;/xref:System.Net.NetworkInformation.IPStatus&gt; &lt;/xref:System.Net.NetworkInformation.PingReply&gt; &lt;/xref:System.Net.NetworkInformation.Ping&gt;</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Net.NetworkInformation.PingException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class using the specified message.</source>
          <target state="translated">Initialiseert een nieuw exemplaar van de <bpt id="p1">&lt;xref href="System.Net.NetworkInformation.PingException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klasse met het opgegeven bericht.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT">
          <source>This constructor initializes the &lt;xref:System.Exception.Message%2A&gt; property with the text in the <ph id="ph1">`message`</ph> parameter.</source>
          <target state="translated">Deze constructor initialiseert de &lt;xref:System.Exception.Message%2A&gt;eigenschap met de tekst in de <ph id="ph1">`message`</ph> parameter.&lt;/xref:System.Exception.Message%2A&gt;</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT">
          <source>The &lt;xref:System.Exception.InnerException%2A&gt; property is set to <ph id="ph1">`null`</ph>.</source>
          <target state="translated">De &lt;xref:System.Exception.InnerException%2A&gt;eigenschap is ingesteld op <ph id="ph1">`null`</ph>.&lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve">
          <source>A &lt;xref:System.String&gt; that describes the error.</source>
          <target state="translated">Een &lt;xref:System.String&gt;die de fout wordt beschreven.&lt;/xref:System.String&gt;</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Net.NetworkInformation.PingException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with serialized data.</source>
          <target state="translated">Initialiseert een nieuw exemplaar van de <bpt id="p1">&lt;xref href="System.Net.NetworkInformation.PingException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klasse met de geserialiseerde gegevens.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" extradata="MT">
          <source>This constructor is called during deserialization to reconstitute a previously serialized &lt;xref:System.Net.NetworkInformation.PingException&gt; object.</source>
          <target state="translated">Deze constructor is aangeroepen tijdens de deserialisatie voor het opnieuw samenstellen van een eerder geserialiseerde &lt;xref:System.Net.NetworkInformation.PingException&gt;object.&lt;/xref:System.Net.NetworkInformation.PingException&gt;</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve">
          <source>The object that holds the serialized object data.</source>
          <target state="translated">Het object dat de gegevens geserialiseerd object bevat.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve">
          <source>A &lt;xref:System.Runtime.Serialization.StreamingContext&gt; that specifies the contextual information about the source or destination for this serialization.</source>
          <target state="translated">Een &lt;xref:System.Runtime.Serialization.StreamingContext&gt;Hiermee worden de contextuele informatie over de bron of bestemming voor deze serialisatie.&lt;/xref:System.Runtime.Serialization.StreamingContext&gt;</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Net.NetworkInformation.PingException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class using the specified message and inner exception.</source>
          <target state="translated">Initialiseert een nieuw exemplaar van de <bpt id="p1">&lt;xref href="System.Net.NetworkInformation.PingException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klasse met behulp van het opgegeven bericht en de interne uitzondering.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" extradata="MT">
          <source>This constructor initializes the &lt;xref:System.Exception.Message%2A&gt; property with the text in the <ph id="ph1">`message`</ph> parameter, and the &lt;xref:System.Exception.InnerException%2A&gt; property with the <ph id="ph2">`innerException`</ph> parameter.</source>
          <target state="translated">Deze constructor initialiseert de &lt;xref:System.Exception.Message%2A&gt;eigenschap met de tekst in de <ph id="ph1">`message`</ph> parameter, en de &lt;xref:System.Exception.InnerException%2A&gt;eigenschap met de <ph id="ph2">`innerException`</ph> parameter.&lt;/xref:System.Exception.InnerException%2A&gt; &lt;/xref:System.Exception.Message%2A&gt;</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve">
          <source>A &lt;xref:System.String&gt; that describes the error.</source>
          <target state="translated">Een &lt;xref:System.String&gt;die de fout wordt beschreven.&lt;/xref:System.String&gt;</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve">
          <source>The exception that causes the current exception.</source>
          <target state="translated">De uitzondering die ervoor zorgt de huidige uitzondering dat.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>